src/mscorlib/src/System/Diagnostics/Stackframe.cs

```diff
@@ -255,13 +255,13 @@ public override String ToString()
                 {
                     Type[] typars = ((MethodInfo)method).GetGenericArguments();
 
-                    sb.Append("<");
+                    sb.Append('<');
```

Comment:
These changes from const strings to const chars are fine, and they likely shave off some cycles, but they don't actually save allocations, do they?